<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFillDatabase</class>
 <widget class="QWidget" name="QFillDatabase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>340</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox2">
     <property name="title">
      <string>Documents</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="_2">
        <item>
         <widget class="QLabel" name="TextLabel1">
          <property name="text">
           <string>Path to Directory:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KUrlRequester" name="KUDirectory">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="_3">
        <item>
         <widget class="QLabel" name="textLabel1_2">
          <property name="text">
           <string>Default mime-type:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="DefaultMIMEType">
          <property name="text">
           <string>text/xml</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="FixLanguage">
          <property name="text">
           <string>Fix the language</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="Language"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="Subjects">
        <property name="toolTip">
         <string>Define if the documents are categorized in subjects or not.</string>
        </property>
        <property name="statusTip">
         <string>Define if the documents are categorized in subjects or not.</string>
        </property>
        <property name="whatsThis">
         <string>Define if the documents are categorized in subjects or not.</string>
        </property>
        <property name="text">
         <string>Documents are catogorized in subjects?</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="GroupBox1">
     <property name="title">
      <string>Subjects</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The documents may be categorized in subjects (ideal cluster of documents) on a directory basis. The hierarchy of directory can represent a subject tree.&lt;/p&gt;&lt;p&gt;Subjects are interesting for validation purposes.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignJustify|Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="Used">
        <property name="toolTip">
         <string>Specify if the subjects and the documents must be selected by default or not.</string>
        </property>
        <property name="statusTip">
         <string>Specify if the subjects and the documents must be selected by default or not.</string>
        </property>
        <property name="whatsThis">
         <string>Specify if the subjects and the documents must be selected by default or not.</string>
        </property>
        <property name="text">
         <string>All subjects and documents selected by default.</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="_5">
        <item>
         <widget class="QLabel" name="textLabel1">
          <property name="text">
           <string>Maximal depth:</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="KIntSpinBox" name="MaxDepth">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Define the maximal depth of the subject tree. If the depth of the directory hierarchy is greather than this maximal depth, the &amp;quot;lowest&amp;quot; directories are not considered as a specific subjects. If null, it is supposed that each directory corresponds to an unique subject.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="statusTip">
           <string>Define the maximal depth of the subject tree.</string>
          </property>
          <property name="whatsThis">
           <string>Define the maximal depth of the subject tree.</string>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="maxValue" stdset="0">
           <number>2</number>
          </property>
          <property name="minValue" stdset="0">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>251</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="_6">
        <item>
         <widget class="QCheckBox" name="HasParent">
          <property name="toolTip">
           <string>Define if the documents imported are related to an existing parent subject.</string>
          </property>
          <property name="statusTip">
           <string>Define if the documents imported are related to an existing parent subject.</string>
          </property>
          <property name="whatsThis">
           <string>Define if the documents imported are related to an existing parent subject.</string>
          </property>
          <property name="text">
           <string>Has parent subject:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLineEdit" name="ParentName">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ChooseBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Choose</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>HasParent</sender>
   <signal>toggled(bool)</signal>
   <receiver>ParentName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>HasParent</sender>
   <signal>toggled(bool)</signal>
   <receiver>ChooseBtn</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>337</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>FixLanguage</sender>
   <signal>toggled(bool)</signal>
   <receiver>Language</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>41</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>368</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Subjects</sender>
   <signal>toggled(bool)</signal>
   <receiver>GroupBox1</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
